views/js/admin/src/components/common/LoaderOverlay.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <Transition
+    name="mypa__fade"
+    @beforeEnter="shown = true"
+    @afterLeave="shown = false">
+    <div
+      v-show="show"
+      class="h-100 mypa__loader-overlay position-absolute w-100"
+      :class="{
+          'd-flex': show || shown,
+        }">
+      <div class="m-auto ps-spinner" />
+    </div>
+  </Transition>
+</template>
+
+<script lang="ts">
+import { defineComponent } from '@vue/composition-api';
+
+export default defineComponent({
+  name: 'LoaderOverlay',
+  props: {
+    show: {
+      type: Boolean,
+    },
+  },
+
+  data() {
```

Comment:
dit zou net zo goed in een setup kunnen.

Reply:
hehe, ik startte met proberen "zo veel mogelijk" gewoon in options api te houden maar ja...